openrtb_ext/bidders.go

```diff
@@ -73,113 +73,114 @@ func IsBidderNameReserved(name string) bool {
-	BidderVerizonMedia     BidderName = "verizonmedia"
-	BidderVisx             BidderName = "visx"
-	BidderVrtcal           BidderName = "vrtcal"
-	BidderYeahmobi         BidderName = "yeahmobi"
-	BidderYieldlab         BidderName = "yieldlab"
-	BidderYieldmo          BidderName = "yieldmo"
-	BidderYieldone         BidderName = "yieldone"
-	BidderZeroClickFraud   BidderName = "zeroclickfraud"
+	Bidder33Across          BidderName = "33across"
+	BidderAcuityAds         BidderName = "acuityads"
+	BidderAdform            BidderName = "adform"
+	BidderAdgeneration      BidderName = "adgeneration"
+	BidderAdhese            BidderName = "adhese"
+	BidderAdkernel          BidderName = "adkernel"
+	BidderAdkernelAdn       BidderName = "adkernelAdn"
+	BidderAdman             BidderName = "adman"
+	BidderAdmixer           BidderName = "admixer"
+	BidderAdOcean           BidderName = "adocean"
+	BidderAdoppler          BidderName = "adoppler"
+	BidderAdot              BidderName = "adot"
+	BidderAdpone            BidderName = "adpone"
+	BidderAdprime           BidderName = "adprime"
+	BidderAdtarget          BidderName = "adtarget"
+	BidderAdtelligent       BidderName = "adtelligent"
+	BidderAdvangelists      BidderName = "advangelists"
+	BidderAdxcg             BidderName = "adxcg"
+	BidderAdyoulike         BidderName = "adyoulike"
+	BidderAJA               BidderName = "aja"
+	BidderAMX               BidderName = "amx"
+	BidderApplogy           BidderName = "applogy"
+	BidderAppnexus          BidderName = "appnexus"
+	BidderAudienceNetwork   BidderName = "audienceNetwork"
+	BidderAvocet            BidderName = "avocet"
+	BidderBeachfront        BidderName = "beachfront"
+	BidderBeintoo           BidderName = "beintoo"
+	BidderBetween           BidderName = "between"
+	BidderBidmachine        BidderName = "bidmachine"
+	BidderBrightroll        BidderName = "brightroll"
+	BidderColossus          BidderName = "colossus"
+	BidderConnectAd         BidderName = "connectad"
+	BidderConsumable        BidderName = "consumable"
+	BidderConversant        BidderName = "conversant"
+	BidderCpmstar           BidderName = "cpmstar"
+	BidderCriteo            BidderName = "criteo"
+	BidderDatablocks        BidderName = "datablocks"
+	BidderDmx               BidderName = "dmx"
+	BidderDecenterAds       BidderName = "decenterads"
+	BidderDeepintent        BidderName = "deepintent"
+	BidderEmxDigital        BidderName = "emx_digital"
+	BidderEngageBDR         BidderName = "engagebdr"
+	BidderEPlanning         BidderName = "eplanning"
+	BidderEpom              BidderName = "epom"
+	BidderGamma             BidderName = "gamma"
+	BidderGamoshi           BidderName = "gamoshi"
+	BidderGrid              BidderName = "grid"
+	BidderGumGum            BidderName = "gumgum"
+	BidderImprovedigital    BidderName = "improvedigital"
+	BidderInMobi            BidderName = "inmobi"
+	BidderInteractiveOffers BidderName = "interactiveOffers"
```

Comment:
IMHO it's simpler for the bidder name to be all lower case. Please consider:
```
BidderInteractiveOffers BidderName = "interactiveoffers"
```
Thoughts @bsardo?